app/uk/gov/hmrc/helptosave/config/microserviceWiring.scala

```diff
@@ -55,7 +60,7 @@ trait WSHttp
 @Singleton
 class WSHttpExtension extends WSHttp with HttpAuditing with ServicesConfig {
 
-  override val hooks: Seq[HttpHook] = Seq(AuditingHook)
+  override val hooks: Seq[HttpHook] = NoneRequired
```

Comment:
does this mean we're not doing auditing in the backend?

Reply:
yeah, auditing between the service to service calls are not needed